docs/getting-started/basic-concepts.md

```diff
@@ -0,0 +1,26 @@
+# Basic concepts
+
+## What's a Knowledge Box?
+
+A Knowledge Box is a logical container where you will save data of any
+kind.i These data is divided into resources.
+
+A resource is similar to a file but much more powerful. It is usually
+created from a document, but it contain a lot more, as fields,
```

Comment:
```suggestion
created from a document, but it may contain a lot more than the binary, such as multiple fields of various types (layouts, conversations, etc),
```